# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/PWebrtcTCPSocket.ipdl
# Commit: c43589fd0f90
# Full Hash: c43589fd0f90d174eb185b563406fd0a2e6ea712
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Update OnConnected callbacks to specify what type of proxying (if any) is being used, and implement IsProxied based on this information. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45186
# ==============================================================================

diff -r 940d7f787bea -r c43589fd0f90 media/mtransport/ipc/PWebrtcTCPSocket.ipdl
--- a/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:18:15 2019 +0000
+++ b/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:18:33 2019 +0000
@@ -32,7 +32,7 @@
 
 child:
   async OnClose(nsresult aReason);
-  async OnConnected();
+  async OnConnected(nsCString aProxyType);
   async OnRead(uint8_t[] aReadData);
 
   async __delete__();